### PR TITLE
Update README.md based on review result.

### PR DESCRIPTION
**Pull request by riversyang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,25 +67,66 @@ This contract has to be initialized with the following parameters:
...
 
 This action needs the following parameters:
 
 * `appchain_id`: The unique identity in Octopus Network. It cannot be duplicated with any other registered appchain.
 * `website_url`: The url of the official website of the appchain.
-* `github_address`: The address of the github repository of the appchain, if it is an open-source project.
-* `github_release`: The release vesion of the github repository of the appchain, if it is an open-source project.
+* `github_address`: The address of the github repository of the appchain.
+* `github_release`: The release vesion of the github repository of the appchain.
 * `commit_id`: The commit id of source code of the github repository of the appchain, if it is an open-source project.
```
Comment (en) [inline README.md:92]: ~if it is an open-source project.~

File: README.md [hunk 2]
```diff
@@ -7,17 +7,19 @@ Contents:
 * [Teminology](#terminology)
```
Comment (en) [inline README.md:7]: Terminology

File: README.md [hunk 3]
```diff
@@ -63,25 +67,66 @@ This contract has to be initialized with the following parameters:
...
+* The `sender` must be the `owner`.
+* The `appchain state` of `appchain basedata` corresponding to `appchain_id` must be `registered`.
+
+The `appchain state` of `appchain basedata` corresponding to `appchain_id` is set to `auditing`.
+
+Generate log: `Appchain <appchain_id> starts auditing.`
+
+### Register an appchain
+
+The `appchain owner` can transfer a certain amount (not less than `minimum register deposit`) of OCT token to this contract by calling function `ft_transfer_call` of `oct_token_contract`. And the calling param `msg` MUST be `register appchain <appchain_id>,<website_url>,<github_address>,<github_release>,<commit_id>,<contact_email>`.
```
Comment (en) [inline README.md:121]: Will anyone deposit more than the minimum amount?

File: README.md [hunk 4]
```diff
@@ -34,23 +36,25 @@ Contents:
...
 * `appchain anchor`: A NEAR contract which is deployed in a subaccount of the account of this contract. It is in charge of managing the necessary data of an appchain on NEAR protocol, providing security and interoperability for the appchain. The anchor contracts are controlled by the `owner` (Octopus DAO) too, and the [octopus-appchain-anchor](https://github.com/octopus-network/octopus-appchain-anchor) is the standard implementation provided by Octopus Core Team.
-* `octopus relayer`: A standalone service which will monitor the state change of the validators of an appchain and facts happened on an appchain. It relays messages between an appchain and corresponding `appchain anchor`.
-* `appchain owner`: The owner of an appchain, ususally the developer or someone who represent the developer team.
-* `register deposit`: To prevent abuse of audit services, an appchain has to deposit a small amount of OCT token when register.
+* `appchain basedata`: The basedata of an appchain, which contains the following fields:
+  * `appchain owner`: The owner of an appchain, ususally the developer or someone who represent the developer team.
```
Comment (en) [inline README.md:40]: usually

**Review by en — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (riversyang) [reply README.md]: Change it to 'must be equal to'.

**Pull request merged**